Formula/tweak.rb

```diff
@@ -0,0 +1,20 @@
+class Tweak < Formula
+  desc "Command-line, ncurses library based hex editor"
+  homepage "https://www.chiark.greenend.org.uk/~sgtatham/tweak/"
+  url "https://www.chiark.greenend.org.uk/~sgtatham/tweak/tweak-3.02.tar.gz"
+  sha256 "5b4c19b1bf8734d1623e723644b8da58150b882efa9f23bbe797c3922f295a1a"
+
+  depends_on "cmake" => :build
```

Comment:
Unnecessary

Reply:
do you mean the white space is unnecessary? or the url line? or the sha256 line? or the depends on line?

Reply:
The `cmake` line